### PR TITLE
Updates to vuetify 2

### PR DESCRIPTION
**Pull request by kelsos**
Also removes cypress to avoid downloading it on CI since it is not used.

### CODE REVIEW

File: raiden-dapp/src/components/AddressInput.vue [hunk 1]
```diff
@@ -197,11 +197,13 @@ export default class AddressInput extends Mixins(BlockieMixin) {
   }
 }
 
-.address-input /deep/ .v-text-field__details {
+//noinspection CssInvalidPseudoSelector
```
Comment (andrevmatos) [inline raiden-dapp/src/components/AddressInput.vue]: Any way to disable this globally? eslint and friends usually support tweaking individual configs

Comment (andrevmatos) [reply raiden-dapp/src/components/AddressInput.vue]: From [vue's docs](https://vue-loader.vuejs.org/guide/scoped-css.html#deep-selectors), it seems both syntax are supported

Comment (kelsos) [reply raiden-dapp/src/components/AddressInput.vue]: So as soon as I switched from `node-sass` to `sass` as per the vuetify upgrade notes the `/deep/` operator started failing. I tried the `>>>` but it seems to not work at all. The documentation actually says that the `sass` preprocessor might have issues with `>>>`

Comment (kelsos) [reply raiden-dapp/src/components/AddressInput.vue]: Ok it seems that this error was an IDE error, I removed the comment and reduced the severity to warning.

**Comment by kelsos** [comment]
@andrevmatos I actually went through at least once everything. Hopefully I didn't miss anything but I will keep an eye

**Pull request merged**